### DISCUSSION TITLE
How can I open my own custom url instead of default blank tab while running the wxt extension?

### DISCUSSION THREAD

**Discussion by nishu-murmu** [wxt-dev/wxt#1147] [Q&A]
Whenever I run my extension, by default it opens to blank tab
[You can see in the screenshot ](https://img.enacton.com/ShareX/2024/1108/chrome_wlQly7XZhW.png)

How can I open my own custom url while running the extension?

**Comment by 1natsu172** [comment]
Use `startsUrl` in your config runner property.
https://wxt.dev/api/reference/wxt/interfaces/ExtensionRunnerConfig.html#starturls

**Reply by nishu-murmu** [reply]
[Is this correct?](https://img.enacton.com/ShareX/2024/1108/Cursor_IIoLtEGnAW.png) Cause this isn't working.

By default it still opening this page.
https://img.enacton.com/ShareX/2024/1108/brave_nw6w5EkLRV.png

**Reply by 1natsu172** [reply]
That is a bug in upstream and there is nothing we can do about it.
https://github.com/mozilla/web-ext/issues/1979

**Reply by nishu-murmu** [reply]
Bummer!! 
We'll have to wait I suppose for it to be fixed.
